lib/method_source/code_helpers.rb

```diff
@@ -104,15 +104,15 @@ def extract_first_expression(lines, consume=0, &block)
     # @param [Array<String>]  lines
     # @return [String]
     def extract_last_comment(lines)
-      buffer = ""
+      buffer = "".dup
 
       lines.each do |line|
         # Add any line that is a valid ruby comment,
         # but clear as soon as we hit a non comment line.
         if (line =~ /^\s*#/) || (line =~ /^\s*$/)
           buffer << line.lstrip
         else
-          buffer.replace("")
+          buffer.replace("".dup)
```

Comment:
This dup isn't necessary, `replace` will replace the contents of buffer with the provided but won't copy its frozenness. There's also a more appropriate method we can use.

```suggestion
          buffer.clear
```